### DISCUSSION TITLE
@formily/element 缺少FormCollapse

### DISCUSSION THREAD

**Discussion by BlueYeah** [alibaba/formily#2139] [Q&A 2.x] [answered]
在@formily/element的仓库里有FormCollapse的文件。 但实际在项目安装后却没有了FormCollapse文件。
重新卸载和安装也没用。
![1631351453461](https://user-images.githubusercontent.com/18047981/132942804-c4d7388d-33f5-43b3-b77e-85343bb3ad16.jpg)
![image](https://user-images.githubusercontent.com/18047981/132942832-215f1243-9aee-4ff0-9988-5ecb5fff7b69.png)

谢谢!

**Comment by muuyao** [comment, marked as answer]
前两天刚加上，还没有发版本，可以等一下
